website/community/syncs.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 2
+title: "Community Syncs"
```

Comment:
Does "Meetups" sound better than "Syncs"?

Reply:
Meetups have a certain expectation in mind. prefer to not use that. "events" sounds too formal. keeping syncs.

Reply:
or "Community meetings" ?